detox/android/detox/src/full/java/com/wix/detox/adapters/server/DetoxServerAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package com.wix.detox.adapters.server
+
+import android.util.Log
+import com.wix.detox.common.DetoxLog
+
+interface OutboundServerAdapter {
+    fun sendMessage(type: String, payload: Map<String, Any?>, id: Long)
+}
+
+class DetoxServerAdapter(
+        private val actionsDispatcher: DetoxActionsDispatcher,
+        private val detoxServerInfo: DetoxServerInfo,
+        private val readyActionType: String,
+        private val terminationActionType: String)
+    : WebSocketClient.WSEventsHandler, OutboundServerAdapter {
+
+    private val wsClient = WebSocketClient(this)
+
+    fun connect() {
+        wsClient.connectToServer(detoxServerInfo.serverUrl, detoxServerInfo.sessionId)
```

Comment:
Should not we log that we are attempting to connect to the server here.?

Reply:
Done